sds_data_manager/lambda_code/SDSCode/batch_starter.py

```diff
@@ -396,7 +406,7 @@ def lambda_handler(event: dict, context):
                 "version": downstream_data["version"],
             }
 
-            update_status_table(status_params)
+            update_status_table(session, status_params)
```

Comment:
So is the basic idea that this call will throw an error if the session already exists? Does that error have enough information about why this job failed?

Reply:
This one is associated with the first commit where we are passing a single session around rather than grabbing a new session each time we make a request to the status table. This avoids the latency associated with requesting a new connection to the engine.